test/CMakeLists.txt

```diff
@@ -31,10 +31,14 @@ set(tests
         integration.cpp
         libclang_parser.cpp
         parser.cpp
+        preprocessor.cpp
         visitor.cpp)
 
+    message(${CMAKE_CURRENT_LIST_DIR})
+
 add_executable(cppast_test test.cpp test_parser.hpp ${tests})
 target_include_directories(cppast_test PUBLIC ${CMAKE_CURRENT_BINARY_DIR})
+target_include_directories(cppast_test PRIVATE ${CMAKE_CURRENT_LIST_DIR}/../src)
```

Comment:
Why is this necessary?

Reply:
It tests the cppast::detail::preprocess function, which is not exported publicly, just an inner library function.

Reply:
Ah, right.